Bergado_Shamma/script.js

```diff
@@ -0,0 +1,47 @@
+document.addEventListener("DOMContentLoaded", function () {
+  let count = 0;
+  let timerInterval;
+  let timerDisplay = document.getElementById("timer");
+  let countList = document.getElementById("count_list");
+
+  function startTimer() {

```

Comment:
I suggest your startTimer function to make it a guard clause approach. 

function startTimer() {
  if (timerInterval) return; // Guard clause
  
  timerInterval = setInterval(() => {
    count++;
    timerDisplay.textContent = count;
  }, 1000);
}

Reply:
Thank you po!